extensions/pangu-for-raycast/README.md

```diff
@@ -0,0 +1,12 @@
+# Pangu for Raycast
+
+Add spaces between Chinese and English, number or symbols using [https://github.com/vinta/pangu.js]().
+
+![](assets/example.gif)
+
+## Usage
+
+1. (可选) 在Raycast Extensions 中为命令 `pangu` 配置快捷键。
+   (Optional) Configure a shortcut for command `pangu` in Raycast Extensions.
+2. 使用快捷键或 Raycast 命令入口触发该命令，它将格式化屏幕上选择的文本，如果没有选择或者内容微卡，它将格式化剪贴板中的内容，然后粘贴到当前光标位置。
+   Trigger the command using the shortcut or Raycast. The command will format the text you've selected on your screen, or format the content from your clipboard, then pastes it to the current cursor location.
```

Comment:
At the moment, Raycast doesn't support localization and only supports US English.